tests/demo_pkg_inline/build.py

```diff
@@ -1,3 +1,5 @@
+"""PLEASE KEEP THIS FILE PYTHON 2.7 COMPATIBLE"""
```

Comment:
There is no need to use all caps. Please add a reference to the development documentation so you can read about the reasons.